np12.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+
+def remove_single(shape):
```

Comment:
Variable names should be expressive and not generic. 
https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0
 Shape is used twice in this dunction differently.